posts/2022/05/thoughts-on-reviewing-for-accessibility.md

```diff
@@ -0,0 +1,185 @@
+
+Remember how [my last post in this series](https://labs.quansight.org/blog/2021/09/not-a-checklist/) 
+called out accessibility as much more complex than a checklist? True to my 
+sense of humor, this blog post is now a checklist. Irony? I don’t know the 
+meaning of the word.
+
+Okay, okay. But seriously, when I’m not making my own work, much of my time is 
+spent reviewing other people’s work. Whether it’s design files, code 
+contributions, blog posts, documentation, or who-knows-what-this-week, I often 
+find myself asking questions and giving feedback about accessibility in the 
+review process. This has prompted multiple people to ask me what it is I’m 
+considering when I review for accessibility. Enough people have now asked that 
+I’ve decided to write something down.
+
+<!-- TEASER_END -->
+
+I’d like to remind readers everywhere that other people have written similar 
+lists in the past and probably will do so again in the future. This list gets 
+long quickly, so I’ve tried to keep each item short and link out for more information.
+
+Without further ado, this is my non-exhaustive but totally honest checklist 
+for accessibility review.
+
+### My review checklist
+
+#### Text
+
+- All-caps are used only when they are needed (like acronyms), not just 
+for emphasis. (Or they use a text-transform property.)
+- [Text has high enough contrast for its size and weight](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum). Make sure to watch out for yellow, orange, and green especially; they tend to be tricky.
+- [Information is not represented only by color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color). (Adding label text or an icon are my most frequently recommended additions.)
+- [Links have appropriately descriptive names](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)—meaning you know where that link text will take you or why it is there.
+- Are there headers or some kind of division? [Headers must follow a clearly ordered hierarchy](https://www.w3.org/WAI/tutorials/page-structure/headings/). (For example, not skipping from Heading 1 to Heading 4 in HTML-based situations.)
+- Acronyms need to be defined (or at least linked to their source).
+- Is the text written in a style appropriate for the audience? In most cases, it needs to use [plain language](https://www.plainlanguage.gov/guidelines/). 
+- Is any [language being unnecessarily gendered](https://developers.google.com/style/inclusive-documentation#gendered-language)? If it is, replace it. (More on [inclusive language at 18F](https://content-guide.18f.gov/our-style/inclusive-language).)
+- Is necessary gendered language used correctly? (Usually, this means don’t misgender people.)
+- Is there [language that unnecessarily assumes the reader’s ability](https://developers.google.com/style/inclusive-documentation#ableist-language)? (For example, assuming the reader can see with “As you can see…”) If there is, replace it.
```

Comment:
I often do the same thing in conversation! I'm not sure it's considered wrong by everyone, but I think it's roughly the same argument as not using "guys" in english as the descriptor of a group of people who aren't all men. So the idea that we've so normalized an expectation of ability that we apply it beyond it's literal use, but that it also erases our perception of those disabled experiences in the process.

To be clear, I'm not sure everyone in accessibility and/or wider inclusion communities agrees this should be removed. I also think there are cases where removing this bias is prohibitive because it complicates something beyond reason.